.circleci/config.yml

```diff
@@ -391,17 +425,21 @@ workflows:
           requires: [react_build_storybook]
           filters:
             branches: { ignore: /(docs|patternlab\/|react\/|core\/).*/ }
+     # core
+      - core_build_storybook:
+          requires: [react_build_storybook]
```

Comment:
I don't think this should depend on react_build_storybook unless it uses something from it?